pkg/sdk/go/bootstrap.go

```diff
@@ -46,36 +46,41 @@ type ConfigUpdateCertReq struct {
 	CACert     string `json:"ca_cert"`
 }
 
-func (sdk mfSDK) AddBootstrap(token string, cfg BootstrapConfig) (string, error) {
+func (sdk mfSDK) AddBootstrap(token string, cfg BootstrapConfig) (string, errors.SDKError) {
```

Comment:
This is an example of how we can use `errors.CheckError` insetad of `encodeError`:
```suggestion
func (sdk mfSDK) ViewBootstrap(token, id string) (BootstrapConfig, errors.SDKError) {
	url := fmt.Sprintf("%s/%s/%s", sdk.bootstrapURL, configsEndpoint, id)
	req, err := http.NewRequest(http.MethodGet, url, nil)
	if err != nil {
		return BootstrapConfig{}, errors.NewSDKError(err.Error())
	}

	resp, err := sdk.sendRequest(req, token, string(CTJSON))
	if err != nil {
		return BootstrapConfig{}, errors.NewSDKError(err.Error())
	}
	defer resp.Body.Close()

	if err := errors.CheckError(resp, http.StatusOK); err != nil {
		return BootstrapConfig{}, err
	}

	var bc BootstrapConfig
	if err := json.NewDecoder(resp.Body).Decode(&bc); err != nil {
		return BootstrapConfig{}, errors.NewSDKError(err.Error())
	}

	return bc, nil
}
```